app/src/main/res/layout/chat_massage.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
typo in the name of this file, should be: chat_message.xml

Reply:
@aviadud can you please refactor and change the file name to chat_message.xml?

Reply:
see the final commit